allennlp/training/optimizers.py

```diff
@@ -521,14 +521,14 @@ def make_sparse(values):
 
                 else:
                     # Decay the first and second moment running average coefficient
-                    exp_avg.mul_(beta1).add_(1 - beta1, grad)
-                    exp_avg_sq.mul_(beta2).addcmul_(1 - beta2, grad, grad)
+                    exp_avg.mul_(beta1).add_(grad, alpha=1 - beta1)
```

Comment:
Someone may want to double-check these changes.

Reply:
I did. The documentation is dangerously confusing about this, but when I try it in the console I get the same results before and after.

Same for the other two

Reply:
Thanks for confirming this!